pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -2170,6 +2170,12 @@ type ConnectionsSpec struct {
 	// +nullable
 	// +optional
 	Compression *CompressionSpec `json:"compression,omitempty"`
+
+	// Whether to require msgr2 even if compression or encryption are not enabled.
+	// If true, the msgr1 port will be disabled.
+	// Requires a kernel that supports msgr v2 (mainline kernel 5.11 or CentOS 8.4).
+	// +optional
+	RequireMsgrV2 bool `json:"requireMsgrV2,omitempty"`
```

Comment:
should we call this msgrv2 or just msgr2 as per ceph https://docs.ceph.com/en/quincy/dev/msgr2/?

Reply:
+1 on `msgr2`. Rook already calls it `msgr2`

Reply:
Agreed on the consistency, will update it